projects/bp-gallery/src/components/views/location-curating/location-management-helpers.tsx

```diff
@@ -503,7 +505,22 @@ export const useDeleteTag = (
     refetch,
     parentTag
   );
+  const tagPicturesQueryResponse = tagPictures({
+    variables: { tagID: locationTag.id },
+    fetchPolicy: 'no-cache',
+  });
+  const flattenedPictures = useSimplifiedQueryResponseData(tagPicturesQueryResponse.data);
+
   const deleteTag = useCallback(async () => {
+    if (flattenedPictures && flattenedPictures.locationTag.pictures.length) {
```

Comment:
Prefer using an optional chain expression instead, as it's more concise and easier to read
aka
```ts
if (flattenedPictures?.locationTag.pictures.length) { 
```